Testshell/main.cpp

```diff
@@ -335,6 +337,58 @@ TEST_F(TestShellFixture, TestScript1ShortcutSUCCESS) {
   EXPECT_EQ("Script 1 executed successfully.\n", output);
 }
 
+TEST_F(TestShellFixture, TestScript2FAIL) {
+  Command cmd{TEST_SCRIPT_2_FULLNAME, {"0xAAAABBBB"}};
+
+  EXPECT_CALL(ssd, write(_, 0xAAAABBBB)).Times(AtLeast(1));
+  EXPECT_CALL(ssd, getResult()).WillRepeatedly(Return("0xFFFFFFFF"));
+
+  CaptureStdout();
+  ts.executeCommand(cmd);
+  std::string output = GetCapturedStdout();
+
+  EXPECT_EQ("Script 2 execution failed.\n", output);
+}
+
+TEST_F(TestShellFixture, TestScript2ShortcutFAIL) {
+  Command cmd{TEST_SCRIPT_2_SHORTCUT, {"0xAAAABBBB"}};
+
+  EXPECT_CALL(ssd, write(_, 0xAAAABBBB)).Times(AtLeast(1));
+  EXPECT_CALL(ssd, getResult()).WillRepeatedly(Return("0xFFFFFFFF"));
+
+  CaptureStdout();
+  ts.executeCommand(cmd);
+  std::string output = GetCapturedStdout();
+
+  EXPECT_EQ("Script 2 execution failed.\n", output);
+}
```

Comment:
추후에는 스크립트 도중에 어디서 fail 이 되었는 지까지 출력해야 할 수도 있을 거 같습니다.
Fail 문구를 output 이 포함하는 지로 바꾸는 건 어떤가요?

```suggestion
  EXPECT_EQ( output.find("SSD Test Shell - Simple and Powerful SSD Testing Tool") !=
    std::string::npos);
}
```